src/components/01-branding/02-typography/navigation/navigation-default.njk

```diff
@@ -0,0 +1,62 @@
+<table class="usa-table">
```

Comment:
Please use BEM classes.

These classes should just be used in their respective components